webpack.playground.config.js

```diff
@@ -55,8 +57,9 @@ module.exports = {
       // webpack-dev-server v2 only watch contentbase on root level
```

Comment:
This comment already explains exactly why the changes weren't being detected.
The added helper function helps set the paths explicitly.
Alternatively, we could hardcode the paths if they don't change frequently.

Reply:
wow... this is simply. 🤦 
I think we can hard code the path, which are not changed frequently.

Reply:
Will do. Did you confirm it's reloading the way you want?

Reply:
Did you verify if the mock data has been reflected after webpack reload?

Reply:
@haishengwu-okta Oops. Missed your edit. I just pushed the hardcoded version.
I can revert if you think the helper version is better, but I'd say the hardcoded paths makes it clear that subdirectories need to be added explicitly. If anything I should add a note to that section explaining that devs should update the webpack config with new paths if they're adding new folders.

Reply:
> Did you verify if the mock data has been reflected after webpack reload?

Yes, but I'll check again.

Reply:
looks like it will not reload the mock server

Reply:
IMHO the real issue is that these changes to `responseConfig.json` are tracked in git.

The **better** setup would be to have a sensible/common default for `responseConfig.json`, then have untracked "local" (i.e., personal or team) variations so devs aren't clobbering each other's test flows.

The **ideal** setup might be if it's a completely dynamic response that either...
1. uses magic test values to trigger specific responses (documentation required of course)
2. allows you to configure the responses via CLI (wishful)

Reply:
> looks like it will not reload the mock server

Looking into it

Reply:
yeah, that's kind of real issue and #2 sounds good to me (CLI, or Web). 
but I'd assume it would take extra effort.
The ultimate goal is to make dev easier to use different set up of mocks.

#1 would require set up too many combinations.

Reply:
> looks like it will not reload the mock server

Okay. Yes, definitely jumped the gun on this (I must've restarted the dev-server manually without realizing). The routes are registered in the `before` hook in webpack-dev-server which is run once at the start, and not triggered for re-compilations. I'll investigate ways to make it happen on file changes.